File: app/app/src/main/app/src/main/res/layout/app/src/main/res/values/app/src/main/java/com/lokaldto/app/src/main/java/com/lokaldto/app/MainActivity.java
package com.lokaldto.app;

import android.app.Activity;
import android.os.Bundle;
import android.graphics.Color;
import android.view.Gravity;
import android.widget.TextView;

public class MainActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        TextView text = new TextView(this);

        text.setText("LoKalDiTo TEST 4");
        text.setTextSize(28);
        text.setTextColor(Color.WHITE);
        text.setBackgroundColor(Color.rgb(8, 107, 22));
        text.setGravity(Gravity.CENTER);

        setContentView(text);
    }
}

File: app/app/src/main/app/src/main/res/layout/app/src/main/res/values/app/src/main/java/com/lokaldto/app/MainActivity.kt
package com.lokaldto.app

import android.app.Activity
import android.os.Bundle
import android.graphics.Color
import android.view.Gravity
import android.widget.TextView

class MainActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        val screen = TextView(this)

        screen.text = "LoKalDiTo is working!"
        screen.textSize = 28f
        screen.setTextColor(Color.WHITE)
        screen.setBackgroundColor(Color.rgb(8, 107, 22))
        screen.gravity = Gravity.CENTER

        setContentView(screen)
    }
}
